plugins/reporters/opossum/src/test/kotlin/OpossumReporterTest.kt

```diff
@@ -130,22 +133,24 @@ class OpossumReporterTest : WordSpec({
         }
 
         "create a file list that contains files from other lists" {
-            opossumInput.pathToSignal.keys.forAll { path -> fileList shouldContain resolvePath(path) }
+            opossumInput.resourcesToAttributions.keys.forAll { path ->
+                val pathWithoutTrailingSlash = if (path != "/") path.removeSuffix("/") else "/"
+                fileList shouldContain resolvePath(pathWithoutTrailingSlash)
+            }
 
             opossumInput.attributionBreakpoints.map { it.replace(Regex("/$"), "") }.forAll { path ->
                 fileList shouldContain resolvePath(path)
             }
-
-            opossumInput.packageToRoot.values.forAll { levelForPath ->
-                levelForPath.keys.forAll { path ->
-                    fileList shouldContain resolvePath(path)
-                }
-            }
```

Comment:
Deleting this because otherwise I would have to write a separate test for `OpossumInputCreator` (and make the property `packageToRoot` non private, I guess).

But it seems like this test does not add much value. The same file paths are in `resourcesToAttributions`, via `addSignal` and later processing of `pathToSignals`. And we have a test for paths in `resourcesToAttributions` against the same list.